packages/react-tabs/src/components/Tab/useTabStyles.ts

```diff
@@ -75,62 +144,133 @@ const useFocusStyles = makeStyles({
   }),
 });
 
-/**
- * Indicator styles for the root slot when horizontal.
- */
-const useHorizontalIndicatorStyles = makeStyles({
+/** Indicator styles for when pending selection */
+const usePendingIndicatorStyles = makeStyles({
   base: {
-    ':after': {
+    ':before': {
```

Comment:
missed this before, but this should probably only be applied on `:hover` for the same reason as the `:after` styles -- right now it's making all tabs look selected all the time in WHCM 😅